app/scripts/views/index.js

```diff
@@ -45,6 +46,13 @@ define(function (require, exports, module) {
       } else if (this.isInEmailFirstExperimentGroup('treatment') || action === 'email') {
         // let's the router know to use the email-first signin/signup page
         this.notifier.trigger('email-first-flow');
+        const email = this.relier.get('email');
+        if (email) {
+          const TEST_REQUEST_DELAY_MS = this.broker.isAutomatedBrowser() ? 500 : 0;
```

Comment:
> Do we have anything like that?

We do! https://github.com/mozilla/fxa-content-server/blob/ce4b91116a0cb80f96ca8dd7c9e41c39e07a2afa/app/tests/spec/models/auth_brokers/base.js#L314-L326